.gitignore

```diff
@@ -64,7 +64,7 @@ distribute-*.tar.gz
 .ropeproject
 
 # ipython/jupyter notebooks and checkpoints
-*.ipynb
+#*.ipynb
```

Comment:
This was introduced to enforce the use of `pre-commit` hooks. Could you please revert this, and enable the hooks, which should then automatically convert your Jupyter notebooks as python scripts with JupyText ?

Reply:
I changed the .gitignore and enable commit hooks